src/components/list/list.scss

```diff
@@ -138,11 +138,13 @@ md-list-item {
       align-items: center;
       justify-content: flex-start;
 
-      padding: 0;
+      padding: 0 16px;
       margin: 0;
 
+      // The button should not inherit the parents background color.
+      background-color: initial;
```

Comment:
No, that's just our button wrap, it's just executing the ripple and the click attributes. It should not inherit the background, otherwise the text content will be invisible.

http://codepen.io/anon/pen/dMpWpQ